vk-video/src/vulkan_decoder/session_resources.rs

```diff
@@ -27,6 +27,7 @@ pub(super) struct VideoSessionResources<'a> {
     pub(crate) decoding_images: DecodingImages<'a>,
     pub(crate) sps: HashMap<u8, SeqParameterSet>,
     pub(crate) pps: HashMap<(u8, u8), PicParameterSet>,
+    pub(crate) sps_needing_reset: HashMap<u8, SeqParameterSet>,
```

Comment:
I'm not sure if I should handle `pps` in similar manner. I tried implementing it both ways and it didn't really made any difference.